src/components/MainNavBar/MainNavBar.tsx

```diff
@@ -14,8 +14,29 @@ import { useOktaAuth } from "@okta/okta-react";
 
 import UserProfile from "./UserProfile";
 import UserAvatar from "./UserAvatar";
+import UMLSDialog from "./ULMSDialog";
+import { Toast } from "@madie/madie-design-system/dist/react";
+
+import "./MainNavBar.scss";
 
 const MainNavBar = () => {
+  // dialog and toast utilities
+  const [TGT, setTGT] = useState<string>(""); //logged in url used to make requests
+  const [dOpen, setDOpen] = useState<boolean>(false);
```

Comment:
`dOpen` is this a typo or intentional(some funky term like Toast)?

Reply:
Intentional short for dialog. It's the boolean for toast being put on the DOM